TurkishFilter/sections/specific.txt

```diff
@@ -94,6 +94,8 @@ marketdergisi.com##.trend-pst > a[href] > img
 08haber.com##.reklam4
 08haber.com###sag > .s5
 08haber.com###sag > .s > img[src^="/images/372x"]
+yeminlisozluk.com##a[href="http://www.tercumex.com.tr/"]
+yeminlisozluk.com###haberdetay > div[style="float: left; width: 100%"]
```

Comment:
this rule breaks the page

Reply:
you mean this?
![this?](https://user-images.githubusercontent.com/45171506/51404969-04a44780-1b66-11e9-92ef-0c02a77a61fd.png)

I guess, it's build-in third-party promo (or whatever), so I blocked it on purpose

Reply:
ok